poja-custom-java-env-vars.txt

```diff
@@ -0,0 +1,2 @@
+AWS_S3_BUCKET=dummy
+AWS_SES_SOURCE=dummy
```

Comment:
custom java env vars should not have dummy values, please contact an OPS for help

Reply:
done